Classes/GHKit.h

```diff
@@ -5,7 +5,7 @@
 //
 //  Permission is hereby granted, free of charge, to any person
 //  obtaining a copy of this software and associated documentation
-//  files (the "Software"), to deal in the Software without
+//  files (the <GHKit/Software>), to deal in the Software without
```

Comment:
Can you double check your find and replace in this file? I don't think changes like this one were intentional.

Reply:
The deed has been done